### PR TITLE
BugFix: Add information about task inputs

### PR DESCRIPTION
**Pull request by zhen0**
image
Screenshot 2023-02-01 at 12 00 33 PM

The current task run page has a misleading tab called "task inputs".  This adds a modal to explain its purpose. 
Closes https://github.com/PrefectHQ/prefect/issues/7547 and #6627 

@madkinsz - for comment here.  I'd welcome your input on the explanation.  I'd prefer not to remove the tab completely in case it was useful to some users. 

### Example

 
 The UI task run page has a Task Input tab that causes confusion 

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

**Comment by madkinsz** [comment]
@zhen0 we'll also track relationships that aren't from futures, i.e. if you have a task that returns some object then a later task that uses it (and it's a uniquely identifiable object) we'll show the relationship here.

**Comment by zhen0** [comment]
Partner to https://github.com/PrefectHQ/nebula-ui/pull/2308

File: orion-ui/src/pages/TaskRun.vue [hunk 1]
```diff
@@ -13,6 +13,17 @@
         <TaskRunLogs :task-run="taskRun" />
       </template>
 
+      <template v-if="taskRun" #task-inputs-heading>
+        Task inputs
+        <ExtraInfoModal title="Task Inputs">
```
Comment (pleek91) [inline orion-ui/src/pages/TaskRun.vue:18]: I think we need a little spacing here. Icon is a little close

File: orion-ui/src/pages/TaskRun.vue [hunk 2]
```diff
@@ -13,6 +13,17 @@
...
+          <template #default>
+            <p>
+              Task inputs are passed to task runs and can
+              include constants, parameter keys, and other task runs.
```
Comment (madkinsz) [inline orion-ui/src/pages/TaskRun.vue:22]: Hm except right now we are not including constants. It's always a mapping of _parameter keys_ to something the API has been told is to track as an input. Right now, the API is only tracking task run relationships that the client reports when it sees _results from other task runs_.

File: orion-ui/src/pages/TaskRun.vue (hunk 1 above)
Comment (zhen0) [reply orion-ui/src/pages/TaskRun.vue:18]: Sorry!  The screenshot I provided was an old one - we added padding as part of the component in orion design. It now looks like this:

image

Comment (pleek91) [reply orion-ui/src/pages/TaskRun.vue:18]: Nice!

File: orion-ui/src/pages/TaskRun.vue (hunk 2 above)
Comment (zhen0) [reply orion-ui/src/pages/TaskRun.vue:22]: OK.  I may need your input on this one.  Any good suggestions for the exact wording here?

Comment (zhen0) [reply orion-ui/src/pages/TaskRun.vue:22]: Updated and using localization for this to keep updates across oss and cloud simpler. Linked to https://github.com/PrefectHQ/orion-design/pull/1136

**Comment by zhen0** [comment]
Converting to draft until https://github.com/PrefectHQ/orion-design/pull/1136 is released

**Pull request merged**